tests/pytorch/debug/test_log.py

```diff
@@ -164,8 +164,7 @@ def test_numerics(fp8_recipe, feature_dirs):
             num_quantizers=3,
         )
 
-        tensor = torch.zeros(1024, 1024).cuda()
-        tensor[0, :] = 1000
+        tensor = torch.randn(1024, 1024).cuda()
```

Comment:
Should we add some specific zeros here to test the functionality? Or maybe run it through relu or something?